src/base/matrix.rs

```diff
@@ -2370,7 +2370,7 @@ where
 
         for i in 0..num_rows {
             for j in 0..num_columns {
-                // loop counters vary in the matrix size intervals
+                // Loop counters vary in the matrix size intervals
                 // get_unchecked is generally unsafe, but optimizes the code by not performing bound tests
                 resulted_vector.push(unsafe { self.get_unchecked((i, j)) }.clone());
```

Comment:
Please explain why the usage of this *is safe*, how does it respect the safety rules https://doc.rust-lang.org/book/ch19-01-unsafe-rust.html#unsafe-superpowers.